### ISSUE TITLE
2022-06-24 Patch Releases Plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#1699]
## 2.3.5

- #1624 
- #1633 
- #1545 
- #1625 
- #1488
- #1692
- #1682
- #1691
- #1673

## 2.2.13

~_I don't think we want to create any more patch releases for 2.2.x now that Drupal 9.2.x is officially EOL (as of the release of Durpal 9.4.0 earlier this month)._~

- #1624 
- #1633 
- #1625 
- #1488
- #1692
- #1682
- #1673

**Comment by trackleft** [comment]
for 2.3.5

#1625
#1545
#1633
#1624


We should probably include those ☝️ as well
```
e0442772e453d47f7ce96fc2cfc467e6e155a6cf
c0445db3cc7be8c82b1c80506848fe1c749b1dd9
4806eac8cd0aa8b2ef14d5410c74a84ee0fc9ae6
138ed99f9f44232a895c54d414286962d946d4e5
```

**Comment by trackleft** [comment]
For 2.2.x, Urge people to Upgrade ASAP somehow.

**Comment by joeparsons** [comment]
## Pick list

```
git cherry-pick c0445db3cc7be8c82b1c80506848fe1c749b1dd9
git cherry-pick 4806eac8cd0aa8b2ef14d5410c74a84ee0fc9ae6
git cherry-pick 138ed99f9f44232a895c54d414286962d946d4e5
git cherry-pick e0442772e453d47f7ce96fc2cfc467e6e155a6cf
git cherry-pick 01503698ec61c89b90f96c0ef38a9d7119be3401
git cherry-pick a83c08d063dc2d28e158948e017fd489a7eda300
git cherry-pick a6b9002a6d9cb4ff9fd480944876bbb93e3a886f
git cherry-pick d78f3f7ca1c69b73c9f17f312067180eacde32d7
git cherry-pick e8a7e03197422b8efb05a9ddc024b95bee3d52b8
```

## Release notes snippet

`git log 2.3.4..HEAD --pretty='format:- %s'`

```
- Closes #1476, #1520, #1521 Migrate az_remote_video (#1673)
- Closes #1690: Update Drupal core to 9.3.17. (#1691)
- Closes #1557: Removed id hash from CSS selector for Search submit button (#1682)
- Closes #1489: Fix media permission denied for authenticated user role. (#1692)
- Closes #1486 Adds a source migration plugin that filters out old content (#1488)
- Closes #1071, #1621: Use noConflict() method to fix Bootstrap/jQuery UI conflict breaking dialog close icons. (#1625)
- Closes #1544: Add workaround and patch for Paragraphs 1.14 migrate plugin issues. (#1545)
- Closes #1630: Change demo content email addresses to @arizona.edu (#1633)
- Closes #1443 - Add bg-white to accordion's card-body (#1624)
```

**Comment by trackleft** [comment]
Looks good, the noteable difference between the two releases is that one (2.3.5) has a Drupal core update and the other (2.2.13) does not. This is because drupal 9.2 is no longer supported.  

Site maintainers are urged to upgrade to QS 2.3.x ASAP to avoid being left behind.  CWS has tested the upgrade path from versions 2.2.8  and higher into 2.3.x
